samples/test/create-custom-job.test.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+const path = require('path');
+const {assert} = require('chai');
+const {after, describe, it} = require('mocha');
+const uuid = require('uuid').v4;
```

Comment:
nit: I'd usually do `const {v4} = require('uuid')`, as this syntax is closer to the `import` used by TypeScript and ESM.

Reply:
I don't think so. Then we'd either have a bunch of calls to *v4()* all over the place - worse for readability - or we'd have an extra line just to set *const uuid = v4;*.